dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/RelationshipStore.java

```diff
@@ -41,11 +41,13 @@ public interface RelationshipStore extends IdentifiableObjectStore<Relationship>
   String ID = RelationshipStore.class.getName();
 
   List<Relationship> getByTrackedEntity(
-      TrackedEntity trackedEntity, RelationshipQueryParams queryParams);
+      TrackedEntity trackedEntity, RelationshipQueryParams queryParams, boolean includeDeleted);
```

Comment:
`includeDeleted` is a field on `RelationshipQueryParams` which is already passed in as a parameter so there is no need to add an additional parameter 🤔

Reply:
yeah, I know, but then I have to do a larger refactor because I am making a distinction with params null in the count query to avoid `order`. While `deleted` can be used to count

Reply:
@teleivo i did a Store refactor